src/discussions/posts/post/Post.jsx

```diff
@@ -61,17 +71,32 @@ function Post({
 
   return (
     <div className="d-flex flex-column w-100 mw-100" data-testid={`post-${post.id}`}>
-      <DeleteConfirmation
+      <Confirmation
         isOpen={isDeleting}
         title={intl.formatMessage(messages.deletePostTitle)}
         description={intl.formatMessage(messages.deletePostDescription)}
         onClose={hideDeleteConfirmation}
-        onDelete={() => {
+        comfirmAction={() => {
           dispatch(removeThread(post.id));
           history.push('.');
           hideDeleteConfirmation();
```

Comment:
needs to be refactored into function also what is` history.push('.')` is?

Reply:
When a thread is deleted history.push('.') pushes url to previous page which is mypost, posts, etc